src/code/z_demo.c

```diff
@@ -24,31 +48,57 @@ CutsceneStateHandler sCsStateHandlers2[] = {
-    { 0x00E4, 2, 0xB6, 0x020070C0 },       { 0x0225, 2, 0xB7, 0x02004A80 },      { 0x0123, 2, 0xB8, 0x02007990 },
-    { 0x0147, 2, 0xB9, 0x020076D0 },       { 0x0138, 0, 0xBA, 0x02004280 },      { 0x0574, 2, 0x5A, 0x020028E0 },
-    { 0x0538, 2, 0xBB, gForestBarrierCs }, { 0x053C, 2, 0xBC, gWaterBarrierCs }, { 0x0540, 2, 0xBD, gShadowBarrierCs },
-    { 0x0544, 2, 0xBE, gFireBarrierCs },   { 0x0548, 2, 0xBF, gLightBarrierCs }, { 0x054C, 2, 0xAD, gSpiritBarrierCs },
-    { 0x008D, 0, 0xC0, 0x02002BB0 },       { 0x03B4, 0, 0xC7, 0x020055C0 },      { 0x0246, 2, 0xB9, 0x020076D0 },
-    { 0x05E8, 2, 0xC6, 0x0200C9D0 },
+    { 0x0185, AGE_REQ_NONE, 0xA0, gHyruleFieldIntroCs },
+    { 0x013D, AGE_REQ_NONE, 0xA1, gDMTIntroCs },
+    { 0x00DB, AGE_REQ_NONE, 0xA3, gKakarikoVillageIntroCs },
+    { 0x0108, AGE_REQ_NONE, 0xA4, gZorasDomainIntroCs },
+    { 0x0138, AGE_REQ_CHILD, 0xA5, gHyruleCastleIntroCs },
+    { 0x014D, AGE_REQ_NONE, 0xA6, gGoronCityIntroCs },
+    { 0x0053, AGE_REQ_NONE, 0xA7, gTempleOfTimeIntroCs },
+    { 0x0000, AGE_REQ_NONE, 0xA8, gDekuTreeIntroCs },
+    { 0x028A, AGE_REQ_ADULT, 0x18, gHyruleFieldSouthEponaJumpCs },
+    { 0x0292, AGE_REQ_ADULT, 0x18, gHyruleFieldEastEponaJumpCs },
+    { 0x028E, AGE_REQ_ADULT, 0x18, gHyruleFieldWestEponaJumpCs },
+    { 0x0476, AGE_REQ_ADULT, 0x18, gHyruleFieldGateEponaJumpCs },
+    { 0x050F, AGE_REQ_CHILD, 0xA9, gHyruleFieldGetOoTCs },
+    { 0x0102, AGE_REQ_NONE, 0xB1, gLakeHyliaIntroCs },
+    { 0x0117, AGE_REQ_NONE, 0xB2, gGerudoValleyIntroCs },
+    { 0x0129, AGE_REQ_NONE, 0xB3, gGerudoFortressIntroCs },
+    { 0x0157, AGE_REQ_NONE, 0xB4, gLonLonRanchIntroCs },
+    { 0x0028, AGE_REQ_NONE, 0xB5, gJabuJabuIntroCs },
+    { 0x00E4, AGE_REQ_NONE, 0xB6, gGraveyardIntroCs },
+    { 0x0225, AGE_REQ_NONE, 0xB7, gZorasFountainIntroCs },
+    { 0x0123, AGE_REQ_NONE, 0xB8, gDesertColossusIntroCs },
+    { 0x0147, AGE_REQ_NONE, 0xB9, gDeathMountainCraterIntroCs },
+    { 0x0138, AGE_REQ_ADULT, 0xBA, gGanonsCastleIntroCs },
+    { 0x0574, AGE_REQ_NONE, 0x5A, gSunSongGraveSunSongTeachPart2Cs },
+    { 0x0538, AGE_REQ_NONE, 0xBB, gForestBarrierCs },
+    { 0x053C, AGE_REQ_NONE, 0xBC, gWaterBarrierCs },
+    { 0x0540, AGE_REQ_NONE, 0xBD, gShadowBarrierCs },
+    { 0x0544, AGE_REQ_NONE, 0xBE, gFireBarrierCs },
+    { 0x0548, AGE_REQ_NONE, 0xBF, gLightBarrierCs },
+    { 0x054C, AGE_REQ_NONE, 0xAD, gSpiritBarrierCs },
+    { 0x008D, AGE_REQ_ADULT, 0xC0, gSpiritBossNabooruKnuckleIntroCs },
+    { 0x03B4, AGE_REQ_ADULT, 0xC7, gGerudoFortressFirstCaptureCs },
+    { 0x0246, AGE_REQ_NONE, 0xB9, gDeathMountainCraterIntroCs },
+    { 0x05E8, AGE_REQ_NONE, 0xC6, gKokiriForestDekuSproutCs },
 };
 
 // Unused, seems to be an early list of dungeon entrance cutscene locations
-u32 D_8011E304[] = { 0x0200B650, 0x02015600, 0x02014F80, 0x02003F80, 0x02000330, gTowerBarrierCs };
+void* D_8011E304[] = { 0x0200B650, 0x02015600, 0x02014F80, 0x02003F80, 0x02000330, gTowerBarrierCs };
```

Comment:
if this is unused do we really know that this symbol belongs to these addresses? without context of what scenes this is used under, we cant really tell right

Reply:
Probably not no. But I just decided to leave it there since it was already set.  I can take it out though.

Reply:
oh, yeah i didnt realize this wasnt changed by you. I pose that question to others then, and wouldnt mind removing it personally